google/cloud/aiplatform/training_jobs.py

```diff
@@ -4138,6 +4146,20 @@ def _model_upload_fail_string(self) -> str:
             "Model."
         )
 
+    @property
+    def evaluated_data_items_bigquery_uri(self) -> Optional[str]:
+        """BigQuery location of exported evaluated examples from the Training Job"""
+
+        self._assert_gca_resource_is_available()
+
+        try:
+            metadata = self._gca_resource.training_task_metadata
+            return metadata["evaluatedDataItemsBigqueryUri"]
+        except (AttributeError, KeyError):
```

Comment:
Why both `AttributeError` and `KeyError`?

Reply:
Corrected.